app/services/school_data_importer.rb

```diff
@@ -12,9 +12,10 @@ def initialize(logger, start_year = Time.zone.now.year)
     @start_year = start_year
   end
 
-  # TODO: Register and Partner 147: Figure out how exactly we want to handle school updating
   def run
-    CSV.foreach(schools_data_file.path, headers: true, encoding: "ISO-8859-1:UTF-8") do |row|
+    files = school_data_files
+
+    CSV.foreach(files["ecf_tech.csv"].path, headers: true, encoding: "ISO-8859-1:UTF-8") do |row|
```

Comment:
Are there other files in there? What's in them?

Reply:
Yes. There's the groups file, the group links file, and the links file. Contents per https://www.get-information-schools.service.gov.uk/Downloads. Will be using the groups file later to set up MATs